.github/workflows/build.yml

```diff
@@ -86,45 +86,45 @@ jobs:
       with:
         path: 'flambda_backend'
 
-    - name: Cache OCaml 4.12 and dune
+    - name: Cache OCaml 4.14 and dune
       uses: actions/cache@v1
       id: cache
       with:
-        path: ${{ github.workspace }}/ocaml-412/_install
-        key: ${{ matrix.os }}-cache-ocaml-412-dune-341
+        path: ${{ github.workspace }}/ocaml-414/_install
+        key: ${{ matrix.os }}-cache-ocaml-414-dune-361
 
-    - name: Checkout OCaml 4.12
+    - name: Checkout OCaml 4.14
       uses: actions/checkout@master
       if: steps.cache.outputs.cache-hit != 'true'
       with:
         repository: 'ocaml/ocaml'
-        path: 'ocaml-412'
-        ref: '4.12'
+        path: 'ocaml-414'
+        ref: '4.14'
 
-    - name: Build OCaml 4.12
+    - name: Build OCaml 4.14
       if: steps.cache.outputs.cache-hit != 'true'
-      working-directory: ocaml-412
+      working-directory: ocaml-414
       run: |
-        ./configure --prefix=$GITHUB_WORKSPACE/ocaml-412/_install
+        ./configure --prefix=$GITHUB_WORKSPACE/ocaml-414/_install
         make -j $J world.opt
         make install
         # Remove unneeded parts to shrink cache file
-        rm -rf $GITHUB_WORKSPACE/ocaml-412/_install/{lib/ocaml/compiler-libs,lib/ocaml/expunge,bin/*.byte}
+        rm -rf $GITHUB_WORKSPACE/ocaml-414/_install/{lib/ocaml/compiler-libs,lib/ocaml/expunge,bin/*.byte}
 
     - name: Checkout dune github repo
       uses: actions/checkout@master
       if: steps.cache.outputs.cache-hit != 'true'
       with:
         repository: 'ocaml/dune'
-        ref: '3.4.1'
+        ref: '3.6.1'
         path: 'dune'
 
     - name: Build dune
       working-directory: dune
       if: steps.cache.outputs.cache-hit != 'true'
       run: |
-        PATH=$GITHUB_WORKSPACE/ocaml-412/_install/bin:$PATH make release
-        cp dune.exe $GITHUB_WORKSPACE/ocaml-412/_install/bin/dune
+        PATH=$GITHUB_WORKSPACE/ocaml-414/_install/bin:$PATH make release
+        cp _boot/dune.exe $GITHUB_WORKSPACE/ocaml-414/_install/bin/dune
```

Comment:
(I am surprised it comes from `_boot`
rather than `_build`, but the checksums
are the same on my box.)

Reply:
The wrapper script `dune.exe` (seems to be a recent addition?) runs the one in `_boot`, so I just copied that.